.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -0,0 +1,67 @@
+name: Issue
+description: Report an issue with one of the samples in this repository. 
+labels: ["needs triage"]
+body:
+  - type: checkboxes
+    attributes:
+      label: IMPORTANT
+      description: Verify the following requirements have been met before opening an issue.
+      options:
+        - label: My question is related to the samples and content within this repository. For all other issues, please post your question in the [community forum](https://community.esri.com/t5/arcgis-api-for-javascript/ct-p/arcgis-api-for-javascript) or open a ticket with [Esri Technical Support](https://support.esri.com/).
+          required: true
+        - label: I have [checked for existing issues](https://github.com/Esri/jsapi-resources/issues) to avoid duplicates. If someone has already opened an issue for what you are experiencing, please add a 👍 reaction to the existing issue instead of creating a new one.
+          required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Behavior
```

Comment:
Can/should we use "Sentence case" for all titles?

Reply:
Yep, done.